src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -65,14 +76,9 @@ export class TSConfigReader extends OptionsComponent {
      * @param fileName  The absolute path and file name of the tsconfig file.
      */
     load(event: DiscoverEvent, fileName: string) {
-        if (!FS.existsSync(fileName)) {
-            event.addError('The tsconfig file %s does not exist.', fileName);
-            return;
-        }
-
         const { config } = ts.readConfigFile(fileName, ts.sys.readFile);
         if (config === undefined) {
-            event.addError('The tsconfig file %s does not contain valid JSON.', fileName);
+            event.addError('No valid tsconfig file found.', fileName);
```

Comment:
I changed the text since it now is used when there is no file as well as an invalid file

Reply:
So the text `The tsconfig file %s does not contain valid JSON` is used for both?